cloud/user/user.go

```diff
@@ -225,3 +240,227 @@ func ListOrgUsers(out io.Writer, client astrocore.CoreClient) error {
 	table.Print(out)
 	return nil
 }
+
+func AddWorkspaceUser(email, role, workspace string, out io.Writer, client astrocore.CoreClient) error {
+	var userID string
+	err := IsWorkspaceRoleValid(role)
+	if err != nil {
+		return err
+	}
+	ctx, err := context.GetCurrentContext()
+	if err != nil {
+		return err
+	}
+	if ctx.OrganizationShortName == "" {
+		return ErrNoShortName
+	}
+	if workspace == "" {
+		workspace = ctx.Workspace
+	}
+	// Get all org users. Setting limit to 1000 for now
+	users, err := GetOrgUsers(client)
+	if err != nil {
+		return err
+	}
+	if email == "" {
+		user, err := selectUser(users, false)
+		userID = user.Id
+		email = user.Username
+		if err != nil {
+			return err
+		}
+	} else {
+		for i := range users {
+			if users[i].Username == email {
```

Comment:
Does case matter? Or should we lowercase `users[i].Username` before comparing it to `email`?

Reply:
I think we should lowercase email but from what Iv seen everything in users[i].Username is already lower cased. I think the API does this when the user signs up